sdk/cost-management/Azure.ResourceManager.CostManagement/src/Customization/Models/CostManagementExportData.Serialization.cs

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using Azure;
+using Azure.Core;
+using Azure.ResourceManager.CostManagement.Models;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.CostManagement
+{
+    public partial class CostManagementExportData : IUtf8JsonSerializable
+    {
+        internal static CostManagementExportData DeserializeCostManagementExportData(JsonElement element)
+        {
+            if (element.ValueKind == JsonValueKind.Null)
+            {
+                return null;
+            }
+            Optional<ETag> eTag = default;
+            ResourceIdentifier id = default;
+            string name = default;
+            ResourceType type = default;
+            Optional<ResourceManager.Models.SystemData> systemData = default;
+            Optional<ExportFormatType> format = default;
+            Optional<ExportDeliveryInfo> deliveryInfo = default;
+            Optional<ExportDefinition> definition = default;
+            Optional<ExportExecutionListResult> runHistory = default;
+            Optional<bool> partitionData = default;
+            Optional<DateTimeOffset> nextRunTimeEstimate = default;
+            Optional<ExportSchedule> schedule = default;
+            foreach (var property in element.EnumerateObject())
+            {
+                if (property.NameEquals("eTag"u8))
+                {
+                    if (property.Value.ValueKind == JsonValueKind.Null)
+                    {
+                        continue;
+                    }
+                    eTag = new ETag(property.Value.GetString());
+                    continue;
+                }
+                if (property.NameEquals("id"u8))
+                {
+                    id = property.Value.GetString().StartsWith("/") ? new ResourceIdentifier(property.Value.GetString()) : new ResourceIdentifier($"/{property.Value.GetString()}");
```

Comment:
Current existing operations just have only one occurred `ID Validator issue`(https://github.com/Azure/azure-rest-api-specs/issues/23704), temporary use customization workaround

Reply:
Only this id is not start with "/" ?

Reply:
Oh, I check the specs and found some operations still have ID validator issues but the test does not cover them. I will double check them.